public/app/plugins/panel/stat/StatPanel.tsx

```diff
@@ -46,9 +44,19 @@ export class StatPanel extends PureComponent<PanelProps<StatPanelOptions>> {
         sparkline.highlightIndex = sparkline.data.length - 1;
       }
     }
+    const linksSupplier = value.view.dataFrame.fields[value.colIndex].getDataLinksSupplier;
 
     return (
-      <DataLinksContextMenu links={getFieldLinksSupplier(value)}>
+      <DataLinksContextMenu
+        links={
+          linksSupplier
+            ? linksSupplier({
+                calculatedValue: value,
+                valueRowIndex: value.rowIndex,
+              })
+            : null
+        }
+      >
```

Comment:
nit: I've seen this code snippet 3 times in this PR. Maybe we're missing an abstraction here, or maybe the api should change?

Reply:
Good point!